sql/core/src/test/scala/org/apache/spark/sql/CollationStringExpressionsSuite.scala

```diff
@@ -89,6 +89,73 @@ class CollationStringExpressionsSuite
     testRepeat("UNICODE_CI", 3, "abc", 2)
   }
 
+  test("Levenshtein expressions with collation") {
+    def prepareLevenshtein(
+        left: String,
+        right: String,
+        collation: String): Levenshtein = {
+      val collationId = CollationFactory.collationNameToId(collation)
+      val leftLit = Literal.create(left, StringType(collationId))
+      val rightLit = Literal.create(right, StringType(collationId))
+      Levenshtein(leftLit, rightLit)
+    }
+
+    Seq(
+      // scalastyle:off nonascii
```

Comment:
there's a good chance we will use nonascii characters a lot in this file, what about disabling this for the entire class/file?

Reply:
Wrapped the class with `scalastyle:off nonascii`.